From d953f5a40428a7593d840a7a0ae7ad67f8da6ed4 Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Tue, 15 Feb 2022 08:10:48 -0800
Subject: gitignore: cleanup stale entries (#37336)

Aside from not being relevant anymore, if you happen to have these old
test files around (which are no longer generated), the tests will fail.
So make it obvious they exist and let `git clean` remove them.
---
 .gitignore | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/.gitignore b/.gitignore
index d157775783..7b6bc33a97 100644
--- a/.gitignore
+++ b/.gitignore
@@ -17,10 +17,6 @@ tools/vagrant/.vagrant/
 *.bak
 # python artifacts
 *.pyc
-# Vm setup generated files
-cluster.env
-sidecar.env
-kubedns
 # pilot
 pilot/pkg/kube/config
 pilot/pkg/proxy/envoy/envoy
-- 
2.35.3

